doc/en/example/reportingdemo.rst

```diff
@@ -289,6 +289,36 @@ get on the terminal - we are working on that):
     E         ?           ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
     failure_demo.py:99: AssertionError
+    ______________ TestSpecialisedExplanations.test_eq_dataclass _______________
+
+    self = <failure_demo.TestSpecialisedExplanations object at 0xdeadbeef>
+
+        def test_eq_dataclass(self):
+    >       from dataclasses import dataclass
```

Comment:
Gotta fix this

Reply:
hmmm, should we gen docs using py37 now?